### PR TITLE
Updating down stream packages (that use raygun4net )

### PR DESCRIPTION
**Pull request by darcythomas**
Updating down stream packages 

Also updated:

1. package number
2.  icon,
3.  licence (refence)
4. signed the regular packages signed (so we can deprecate the `<xyz>.Signed` packages later

### CODE REVIEW

**Review by Olwiba — APPROVED** [review]
These changes look good to me

**Review by xenolightning — DISMISSED** [review]
I like it.

Will be nice to sign all nuget assemblies

File: Mindscape.Raygun4Net.WebApi.Signed.nuspec [hunk 1]
```diff
@@ -9,10 +9,9 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <summary>Raygun provider for signed ASP.NET Web API projects</summary>
     <description>Raygun provider for signed ASP.NET Web API projects</description>
-    <iconUrl>https://app.raygun.com/Content/Images/nuget-icon.png</iconUrl>
     <projectUrl>https://github.com/MindscapeHQ/raygun4net</projectUrl>
-    <licenseUrl>https://raw.github.com/MindscapeHQ/raygun4net/master/LICENSE</licenseUrl>
-    <dependencies>
+    <license type="file">LICENSE</license>
+    <icon>128x128-transparent.png</icon>  <dependencies>
```
Comment (JasenPalmer) [inline Mindscape.Raygun4Net.WebApi.Signed.nuspec:14]:
```suggestion
    <icon>128x128-transparent.png</icon>
    <dependencies>
```

**Review by JasenPalmer — APPROVED** [review]
Looks good to me. Just one non-blocking suggestion

**Review by JasenPalmer — APPROVED** [review]
Looks good to me

**Pull request merged**